src/app/global-error.tsx

```
@@ -11,13 +11,18 @@ import { Toaster } from '@/components/ui/toaster'
 import { useEffect } from 'react'
 import { useTranslate } from '@/providers/I18n/client'
 
+import * as Sentry from '@sentry/nextjs'
+import Error from 'next/error'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)